client/auth/controller.js

```diff
@@ -20,16 +22,27 @@ import userFactory from 'lib/user';
 import Main from 'components/main';
 import PulsingDot from 'components/pulsing-dot';
 
-module.exports = {
+export default {
 
-	// Login screen used by the desktop application
-	login: function() {
-		if ( OAuthToken.getToken() ) {
-			page( '/' );
-		} else {
-			ReactDom.render(
-				React.createElement( LoginComponent, {} ),
-				document.getElementById( 'primary' )
+	login: function( context ) {
+		if ( config.isEnabled( 'oauth' ) ) {
+			// hack to keep the blue background for Desktop login
+			document.querySelector( '.layout' ).style.backgroundColor = '#0087be';
```

Comment:
Sure it's not very important but I went that way for a few reasons:
- Modifying the route of the existing oauth login would mean create a new section for it and that would imply a lot of changes such as updating all the css classes which would just make that PR even larger.
- We would need to double check that we didn't break anything in `wp-desktop` which means more time spent reviewing.
- In the end, we would like to merge the 2 login components and only have to maintain one so having them both in the same section will increase the likelihood of this happening.